build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -e
+
+${CARMA_OPT_DIR}/scripts/build_script.sh -p $@
```

Comment:
Is this file needed? What is it for?

Reply:
Yes calls the carma-builds build_script with parameters to generate the debian package. The carma-builds build_script is responsible for calling the appropriate cmake and cpack calls with the appropriate options set.

Reply:
I guess we could replace this file by setting the vscode build task to call this directly but I think the script is useful for abstracting this from the developer and just allowing them to use either the vscode task or script with no parameters set.